### DISCUSSION TITLE
Test cell failing in site maps notebook

### DISCUSSION THREAD

**Discussion by Adriannaluger** [earthlab-education/Earth-Analytics-2023-01-Intro#493] [Q&A06 Vector Data] [answered]
When I get to the point when trying to calculate the lengths of each road in the counties I get an output that looks correct to me (I am using codespaces): 
This is the code that I wrote: 
```python
import geopandas as gpd
import pandas as pd

# Calculate length of each road segment and create a new column 'length'
clipped_roads['length'] = clipped_roads['geometry'].length

# Group by 'NAME' (county name) and sum the road lengths for each county
county_lengths = clipped_roads.groupby('County')['length'].sum().reset_index()

# Rename columns for clarity
county_lengths.columns = ['NAME', 'length']

# Set the float format to avoid scientific notation
pd.options.display.float_format = '{:.4f}'.format

# Print the resulting DataFrame
print(county_lengths.to_string(index=False))
```

This is the output:
```python
Del Norte | 121307.5654
Modoc | 245029.1470
Siskiyou | 472427.9500
```
I added the total lengths of all the roads and it is the same number written in the test cell, which leads me to believe that my numbers are correct. I even tried asking Chat GPT for other ways to run this code and did so successfully and got the exact same answer. 
However, the test cell continues to fail. It gives me the message: 
```python
TypeError: object of type 'int' has no len()
```
What should I try to fix this issue?

**Comment by annettebalmes** [comment, marked as answer]
So close, Adri! The highlighted portion shows what you are missing. The test cells are never intuitive for forgetting to call it at the end, always one of the things I check first!

![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/110803009/938846cf-de68-44eb-bc61-87408880e6da)
